### PR TITLE
Improve build scripts

### PR DESCRIPTION
**Pull request by zliang7**

### CODE REVIEW

File: inference-runtime/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2021 Intel Corporation
+
+cmake_minimum_required(VERSION 3.20)
+
+set(LIBTORCH_URL "https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.11.0%2Bcpu.zip")
+set(LIBPADDLE_URL "https://paddle-inference-lib.bj.bcebos.com/2.3.0/cxx_c/Linux/CPU/gcc5.4_openblas/paddle_inference.tgz")
+set(LIBTENSORFLOW_URL "https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.8.0.tar.gz")
+set(LIBONNXRUNTIME_URL "https://github.com/microsoft/onnxruntime/releases/download/v1.7.0/onnxruntime-linux-x64-1.7.0.tgz")
```
Comment (xwu55) [inline inference-runtime/CMakeLists.txt:8]: Onnx has upgraded to version 1.11.0 in CCAI1.3 release.  
https://github.com/microsoft/onnxruntime/releases/download/v1.11.0/onnxruntime-linux-x64-1.11.0.tgz

Comment (zliang7) [reply inference-runtime/CMakeLists.txt]: updated

**Comment by zliang7** [comment]
Meanwhile, the requirement of cmake version is down to 3.16.

**Comment by xwu55** [comment]
looks good to me

**Pull request merged**